modulefiles/ufs_derecho.gnu.lua

```diff
@@ -0,0 +1,30 @@
+help([[
+loads UFS Model prerequisites for Derecho/GNU
+]])
+
+setenv("LMOD_TMOD_FIND_FIRST","yes")
+prepend_path("MODULEPATH", "/lustre/desc1/scratch/epicufsrt/contrib/modulefiles")
+load("ecflow/5.8.4")
+load("mysql/8.0.33")
+
+setenv("LMOD_TMOD_FIND_FIRST","yes")
+prepend_path("MODULEPATH", "/lustre/desc1/scratch/epicufsrt/contrib/modulefiles_extra")
+prepend_path("MODULEPATH", "/glade/work/epicufsrt/contrib/spack-stack/derecho/spack-stack-1.5.1/envs/unified-env/install/modulefiles/Core")
+
+unload("ncarcompilers")
+stack_gnu_ver=os.getenv("stack_gnu_ver") or "12.2.0"
+load(pathJoin("stack-gcc", stack_gnu_ver))
+
+stack_impi_ver=os.getenv("stack_impi_ver") or "8.1.25"
+load(pathJoin("stack-cray-mpich", stack_impi_ver))
```

Comment:
`stack_impi_ver` environment variable should not be used to overwrite the version of `stack-cray-mpich` module. This is error prone and confusing.